components/FAQPage/index.jsx

```diff
@@ -54,6 +54,47 @@ const faqList = [
           </>
         ),
       },
+      {
+        id: 'olas-is-trying-to-solve',
+        title: 'What is the problem Olas is trying to solve?',
+        desc: (
+          <>
+            <p className="mb-3">
+              The best AI is increasingly agentic but centralized.
+            </p>
+            <p className="mb-3">
+              {' '}
+              Everyone [end users, businesses, governments] wants
+              general-purpose autonomous AI agents to do valuable [health,
+              wealth, wellbeing, all industries, all services] things for them.
+            </p>
+            <p className="mb-3">
+              {' '}
```

Comment:
probably can remove these spaces, maybe were added automatically by linter, free to ignore for now